packages/ui-components/src/components/Map/Table/getMapTableData.js

```diff
@@ -44,15 +43,11 @@ const processData = (serieses, measureData) => {
   });
 };
 
-export const useMapTable = (serieses, measureData) => {
+export const getMapTableData = (serieses, measureData) => {
```

Comment:
@chris-bes - same issue applies here about the use of a hook inside a regular function.